test/helpers.bash

```diff
@@ -558,7 +558,11 @@ function dig() {
     # first arg is container_netns_pid
     # second arg is name
     # third arg is server addr
-    run_in_container_netns "$1" "dig" "+short" "$2" "@$3"
+    if [ ! -z "$4" ]; then
+        run_in_container_netns "$1" "dig" "+short" "$2" "@$3" "$4"
+    else
+        run_in_container_netns "$1" "dig" "+short" "$2" "@$3"
+    fi
```

Comment:
just leave the $4 unquoted then you do not need the if else, unless you ever expect that the type could contain special chars or spaces.

Reply:
Yeah sounds better.